crates/driver/src/domain/competition/mod.rs

```diff
@@ -131,7 +131,8 @@ impl Competition {
         .await
         .is_err()
         {
-            observe::postprocessing_timed_out(&settlements)
+            observe::postprocessing_timed_out(&settlements);
+            notify::postprocessing_timed_out(&self.solver, auction.id())
```

Comment:
All the discarded solution IDs could be sent in the notification, but this change is not backward-compatible since the [`solutionId` field](**url**) `number` type has to be switched to `array`. Wdyt?

Reply:
I think for driver <> solver engine communication we don't yet have to be backwards compatible (soon though) so I think this would still possible

Reply:
Ah, sorry, I referred to the wrong item. I was worried about the legacy solvers notification, but looks like it's not the issue.

Reply:
Legacy solvers can only send a single solution